install-ubuntu-mate.sh

```diff
@@ -75,4 +75,5 @@ log_success_msg "DeskPi PWM Fan Control and Safeshut Service installed successfu
 log_action_msg "System will reboot in 5 seconds to take effect." 
```

Comment:
If commenting out the reboot step (which I think is a good change) then removing this echo indicating a reboot is going to happen would make sense.

Reply:
You are right, I only did a quick and dirty change to notify the owner of the script, and to encurage him/her to rewrite it.
However I have seen no reaction but yours, so they might not actively support this shared project at this moment.